src/vmm/src/devices/virtio/queue.rs

```diff
@@ -823,6 +823,14 @@ mod verification {
         }
     }
 
+    mod stubs {
+        use super::*;
+
+        fn set_avail_event<M: GuestMemory>(_self: &mut Queue, _val: u16, _mem: &M) {
+            // do nothing
+        }
```

Comment:
Could you explain a bit better why this does not make us loose generality? IIUC:

`try_enable_notification` uses `set_avail_event` to set the index of the next available buffer to `Queue::next_avail`, but that value is only meant to be read by the guest. It does not affect the behaviour of `try_enable_notification`.

Please keep me true here :)

Reply:
Yeah, that was exactly my thinking! I left a comment on the actual `#[kani::stub(...)]` where this is used further down, but maybe I should also put something here

Reply:
I'd vote for adding the explanation in the commit message and the stub itself, rather than where we call it.

Reply:
Done :)